app/models/concerns/profile_searching.rb

```diff
@@ -107,5 +113,17 @@ def os_major_version_search(_filter, operator, value)
       benchmarks = Xccdf::Benchmark.os_major_version(value, operator == '=')
       { conditions: benchmark_id.in(benchmarks.pluck(:id)).to_sql }
     end
+
+    def ssg_version_search(_filter, operator, value)
+      benchmark_id = Profile.arel_table[:benchmark_id]
+      benchmarks = operator == '=' &&
+                   Xccdf::Benchmark.where(version: value) ||
```

Comment:
I'm not fully sure, I suspect that within conditions it will query for data to evaluate the condition. If that's true, it would query all fields (as by default), in addition to creating a sub query.

Reply:
In this case, the condition below uses `to_sql`, and this is just a setup for `benchmarks.pluck(:id)` below.